src/main/java/codeu/controller/ServerStartupListener.java

```diff
@@ -28,6 +31,21 @@ public void contextInitialized(ServletContextEvent sce) {
       List<Conversation> conversations = PersistentStorageAgent.getInstance().loadConversations();
       ConversationStore.getInstance().setConversations(conversations);
 
+			Conversation actFeedConversation = PersistentStorageAgent.getInstance().loadActFeedConversation();
+			ConversationStore.getInstance().setActFeedConversation(actFeedConversation);
+			
+			/**
+			 * Checks to see if the actFeedConversation has been set in datastore yet.
+			 * If not, creates a new activityFeedConversation and writes it through to datastore.
+			 * Should only ever happen one time.
+			 */
+
+			if (ConversationStore.getInstance().getActFeedConversation().getId() == null ) {
```

Comment:
Wouldn't this throw you an error if there's no Activity feed yet? For example you make a `getActFeedConversation()`, so if it doesn't exist it will return `null`, then you try to access the ID from a null object and the error will show. Not sure though! Correct me if I'm wrong!

Reply:
Okay perfect, thanks for clearing it out!